### ISSUE TITLE
Load Bundle Crash

### ISSUE THREAD

**Issue by NTFLeo** [4taras4/CountryCode#27]
## Inside
```swift
fileprivate func loadViewFromNib() -> UIView {
        let bundle = Bundle(for: type(of: self))
        let nib = UINib(nibName: String(describing: type(of: self)), bundle: bundle)
        let nibView = nib.instantiate(withOwner: self, options: nil).first as! UIView
        
        return nibView
    }
```
## This error occures:

> *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Could not load NIB in bundle: 'NSBundle </var/containers/Bundle/Application/F064407E-A951-47C1-8C3C-AD1938DD885F/MyApp.app> (loaded)' with name 'CountryView''
> ***

**Comment by 4taras4** [comment]
try to use previous version

**Comment by 4taras4** [comment]
fixed in 1.7
